setup.cfg

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2024 Graz University of Technology.
+#
+# Invenio-Curations is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+[metadata]
+name = invenio-curations
+version = attr: invenio_curations.__version__
+description = "Invenio module for generic and customizable curations."
+long_description = file: README.rst, CHANGES.rst
+keywords = invenio curations
+license = MIT
+author = Graz University of Technology
+author_email = info@tugraz.at
+platforms = any
+url = https://github.com/tu-graz-library/invenio-curations
+classifiers =
+    Development Status :: 5 - Production/Stable
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=3.9
+zip_safe = False
+install_requires =
+    invenio-requests>=4.0.0,<5.0.0
+[options.extras_require]
+tests =
+    pytest-black>=0.3.0
+    invenio-app>=1.3.2,<2.0.0
+    invenio-db[postgresql,mysql]>=1.0.13,<2.0.0
+    pytest-invenio>=2.1.0,<3.0.0
+    sphinx>=4.5.0
+elasticsearch7 =
```

Comment:
in my opinion elasticsearch7 is not more necessary. it has been removed from invenio packages too as i know